package.json

```diff
@@ -1,17 +1,23 @@
 {
   "name": "eslint-config-mavenlint",
-  "version": "1.0.3",
+  "version": "1.1.0",
   "description": "Mavenlink ESLint config",
   "main": "index.js",
   "repository": "https://github.com/mavenlink/mavenlint.git",
   "license": "MIT",
-  "peerDependencies": {
-    "eslint": ">= 3.7.0"
+  "devDependencies": {
+    "eslint": "^3.7.0",
+    "eslint-plugin-import": "^1.13.0",
+    "eslint-plugin-jsx-a11y": "^2.1.0",
+    "eslint-plugin-react": "^6.1.2"
   },
-  "dependencies": {
-    "eslint-config-airbnb": "^10.0.1",
+  "peerDependencies": {
+    "eslint": "^3.7.0",
```

Comment:
Expanding the issue more I believe it's because the syntax causes the most recent version of `eslint` to be installed given

`├─┬ UNMET PEER DEPENDENCY eslint@4.2.0`

when the packages want something older such as

```
├─┬ eslint-config-airbnb@10.0.1 
│ ├── UNMET PEER DEPENDENCY eslint@^3.3.1
```

This may be reconciled in an upcoming PR where I update `eslint` to latest. For now, we might need to stick with the caret. Definitely on my radar to ensure we can switch to the recommendation.